stories/Gallery.stories.mdx

```diff
@@ -3450,6 +3456,9 @@ import { Gallery } from '@wpmedia/engine-theme-sdk';
             expandPhrase={text('Expand phrase', 'Expand')}
             autoplayPhrase={text('Autoplay phrase', 'Autoplay')}
             pausePhrase={text('Pause phrase', 'Pause')}
+            displayTitle={boolean('Display Title', true)}
+            displayCaption={boolean('Display Caption', true)}
+            displayCredits={boolean('Display Credits', true)}
```

Comment:
weird I'm not seeing credits. may be a storybook bug, trying to look
![Screen Shot 2021-12-14 at 09 31 01](https://user-images.githubusercontent.com/5950956/146028598-9449067e-18c0-4d58-8efd-2fd54a95439b.png)

Reply:
the features look like they're working though